app/components/ui/molecules/PeoplePicker.test.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
+import PeoplePicker from './PeoplePicker'
+
+const people = [
+  { id: 1, name: 'Annie' },
+  { id: 2, name: 'Bobby' },
+  { id: 3, name: 'Chastity' },
+  { id: 4, name: 'Dave' },
+]
+
+const makeWrapper = props =>
+  mount(
+    <ThemeProvider theme={theme}>
+      <PeoplePicker
+        // eslint-disable-next-line react/no-children-prop
+        children={innerProps => <PeoplePicker.Body {...innerProps} />}
+        onCancel={jest.fn()}
+        onSubmit={jest.fn()}
+        people={people}
+        {...props}
+      />
+    </ThemeProvider>,
+  )
+
+const getPersonPodButton = (wrapper, index) =>
+  wrapper.find('button[data-test-id="person-pod-toggle"]').at(index)
+
+const expectSelectionLength = (wrapper, length) =>
+  expect(wrapper.find('SelectedItem')).toHaveLength(length)
+
+describe('PeoplePicker', () => {
+  it('calls child render prop', () => {
+    const render = jest.fn(() => null)
+    makeWrapper({ children: render })
+    expect(render).toHaveBeenCalled()
+  })
+
+  it('renders one person pod per person', () => {
+    const wrapper = makeWrapper()
+    const buttons = wrapper.find('PersonPod')
+
+    expect(buttons).toHaveLength(4)
+  })
+
+  it('updates selection', () => {
```

Comment:
Perhaps this could be more specific? Like "clicking the icon adds/removes a selected item from the page"?

Because "updates selection" is actually testing a few things all at once:
- whether the icon changes from `add` -> `remove` -> `add` again
- whether clicking the + button adds a person to the `selected` array
- whether clicking the bin button removes a person from the `selection` array
- whether the appearance of the `SelectedItem`s are correctly connected to the `selection` array

If this test fails, by the name of the description, I would assume this was failing because of the connection between the button click & the selection array. I may not thinking that this test was related to the appearance/disappearance of the `SelectedItem`s

Reply:
Yeah. Initially I was asserting against the internal `selection` state of the component so this made sense. But after having to wrap the whole thing in a `ThemeProvider` I can no longer do that :( Name updated.